head.html

```diff
@@ -5,7 +5,7 @@
 <link rel="stylesheet" href="/styles/styles.css"/>
 <script src="https://cdn.auth0.com/js/auth0-spa-js/2.0/auth0-spa-js.production.js"></script>
 <script src="/scripts/auth0.js" type="module"></script>
-<script defer src='https://static.cloudflareinsights.com/beacon.min.js' data-cf-beacon='{"token": "7ef9e67e566e49269a85c45427652359"}'></script>
-<script src="/cdn-cgi/zaraz/i.js" referrerpolicy="origin"></script>
+<!-- <script defer src='https://static.cloudflareinsights.com/beacon.min.js' data-cf-beacon='{"token": "7ef9e67e566e49269a85c45427652359"}'></script>
```

Comment:
Why did you comment this out ?

Reply:
my bad. was annoyed by the console errors of them on localhost, couldnt find my logs

Reply:
maybe we can add the script conditionally if its on kestrelone.com